autoscale/src/main/resources/schema/app/20230201124421_CB-20769_Modify_the_data_type_of_statusreason_in_history_table.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-20769 Modify the data type of statusreason in history table
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE history ALTER COLUMN status_reason TYPE TEXT;
```

Comment:
Unrelated change ?

Reply:
Character varying type is insufficient when attempting to store longer messages in the history entity resulting in PSQL exceptions. Changed it to `text` type here